spec/cocoapods/generate/podfile_generator_spec.rb

```diff
@@ -649,7 +663,87 @@
           end
         end
 
-        expect(podfile_for_spec.to_yaml).to eq expected.to_yaml
+        expect(podfile_for_specs.to_yaml).to eq expected.to_yaml
+      end
+
+      describe 'single workspace podfile' do
```

Comment:
new test that ensures single workspace podfile generation looks correct, includes 2 podspecs with test specs and one of them with an app spec.